ui/raidboss/data/triggers/o8s.js

```diff
@@ -148,14 +148,14 @@
     {
       id: "O8S Future's End",
       regex: /Kefka starts using Future's End/,
-      regexDe: /Kefka starts using Ende Der Hoffnung/,
+      regexDe: /Kefka starts using Ende Der Zukunft/,
```

Comment:
That's funny that German was so different but then they changed it.  That also has to be confusing though if you've been clearing o8s for weeks and then suddenly the ability names aren't the same anymore.